news/2 Fixes/9685.md

```diff
@@ -0,0 +1 @@
+Only set preferred kernel on exact matches.
```

Comment:
Don't mind this comment. I'm just testing issue: https://github.com/microsoft/vscode/issues/148054

</br>
</br>

More comments after br

</br>

Even more comments.